tests/standalone/e2e/ilab_standalone_rhoai_test.go

```diff
@@ -38,6 +38,15 @@ const (
 	STANDALONE_FILE_PATH        = "../../../standalone/standalone.py"
 )
 
+const (
+	SDG_PIPELINE_FILEPATH = "/usr/share/instructlab/sdg/pipelines/agentic"
```

Comment:
Is there a specific reason to specify this pipeline for SDG?

Reply:
this is a rhelai specific "secret sauce" feature for running sdg, it's required